app/models.py

```diff
@@ -1168,7 +1168,7 @@ class Brief(db.Model):
     clarification_questions = db.relationship(
         "BriefClarificationQuestion",
         order_by="BriefClarificationQuestion.published_at",
-        lazy='joined'
+        lazy='select',
```

Comment:
Ah, you're right, I totally misread that as being part of  the `GET /briefs` endpoint